reference/volumes.html.md.erb

```diff
@@ -106,15 +102,17 @@ Created at: 04 Mar 23 03:32 UTC
 ```
 ## Extend a Volume
 
-[Volumes can be extended](https://fly.io/docs/flyctl/volumes-extend/), but cannot be made smaller. To make a volume larger, find its ID with `fly volumes list`, then use:
+Extend a volume using `fly volumes extend`. Volumes can be extended, or made larger, but can't be made smaller. Refer to [`fly volumes extend`](https://fly.io/docs/flyctl/volumes-extend/) in the [flyctl reference](/docs/flyctl) for usage and options. 
+
+To extend a volume, find its ID with `fly volumes list`, then use `fly volumes extend <volume-id> -s <new-size>`:
 
 ```cmd
-fly volumes extend <volume-id> -s <new-size>
+fly volumes extend vol_od56vjp5pzmvny30 -s 2
 ```
 
-where `<new-size>` is the desired size in GB. 
+where `<new-size>` is the extended size in GB. 
 
-The Machine attached to the target volume must be restarted to allow the file system to be resized. **On legacy Nomad-orchestrated Apps, the instance is automatically restarted, so this step isn't needed.**
+You'll need to restart the Machine attached to the target volume to allow the file system to be resized. On Nomad (V1) Apps, the instance is automatically restarted, so this step isn't needed.
```

Comment:
```suggestion
You'll need to restart the Machine attached to the target volume to allow the file system to be resized. On V1 (Nomad) apps, the instance is automatically restarted, so this step isn't needed.
```

I'm not convinced that this last sentence doesn't need to be bold, or have a callout.

Reply:
@catflydotio I can re-bold it. However, a note might be a better pattern for this kind of thing. Style experts discourage using bold for this scenario. :)